x-pack/plugins/canvas/public/expression_types/arg_types/container_style/appearance_form.js

```diff
@@ -31,17 +32,41 @@ export const AppearanceForm = ({ padding, opacity, overflow, onChange }) => {
   return (
     <EuiFlexGroup gutterSize="s" justify-content="spaceBetween">
       <EuiFlexItem grow={2}>
-        <EuiFormRow label="Padding" compressed>
+        <EuiFormRow
+          label={
+            <FormattedMessage
+              id="xpack.canvas.expressionTypes.containerStyle.appearance.formRowPaddingLabel"
```

Comment:
xpack.canvas.expressionTypes.containerStyle.appearance => xpack.canvas.appearance
for all ids in this file